stable/kube-slack/values.yaml

```diff
@@ -1,9 +1,28 @@
 ## Environment values for the deployment
 envVars:
-  ## URL of your Incoming Webhook in Slack:
+  ## URL of your Incoming Webhook in Slack
+  ## e.g. https://hooks.slack.com/services/Txxxxxxxx/Bxxxxxxxx/XXXXXXXXXXXXXXXXXXXXXXXX
   SLACK_URL: ""
+  ## Override channel to send
+  SLACK_CHANNEL: ""
+  ## URL of HTTP proxy used to connect to Slack
+  SLACK_PROXY: ""
   ## Time to wait after pod becomes not ready before notifying
   NOT_READY_MIN_TIME: "60000"  # in ms
+  ## Set to false to disable alert on pod recovery
+  RECOVERY_ALERT: true
+  ## How often to update in milliseconds
+  TICK_RATE: 15000
+  ## Repeat notification after this many milliseconds has passed after status returned to normal
+  FLOOD_EXPIRE: 60000
```

Comment:
try "60000"

Reply:
quoted `"`,

FYI, it's default value of kube-slack
defined here: https://github.com/wongnai/kube-slack/blame/master/README.md#L106

Reply:
Hummm, interesting, after adding quote `"` for all variables for ConfigMaps, test passed, looks like there's a hidden rule behind.